doc/api/path.markdown

```diff
@@ -94,9 +103,13 @@ Windows examples:
 
     path.isAbsolute('//server')  // true
     path.isAbsolute('C:/foo/..') // true
-    path.isAbsolute('bar\\baz')   // false
+    path.isAbsolute('bar\\baz')  // false
     path.isAbsolute('.')         // false
 
+*Note:* If the path string passed as parameter is a zero-length string,
+        unlike other path module functions, zero-length string will be used
+        as-is and `false` will be returned.
```

Comment:
@Fishrock123 I don't have a Windows machine, but when I tried in REPL, I got `false` for both the versions.

@Trott Thanks for the suggestion man :-) I included a simple test now. PTAL.